src/components/header/header.js

```diff
@@ -26,7 +26,7 @@ const Header = ({ title }) => (
   <AnimatedContainer>
     <Container>
       <Link to="/">
-        <Title as="h1">{title}</Title>
+        <Title as="h1">{}</Title>
```

Comment:
This works but would become `AnimatedContainer` not reusable.

I think it's more like making the "white" section not to use `header.js` or just turning it into another components